### ISSUE TITLE
Debian: Install "ogar-linux-script.sh" not working, Fix inside

### ISSUE THREAD

**Issue by SimonOrJ** [OgarProject/Ogar#597]
`ogar-linux-script.sh:205:1`
```
case "$1" in
        install)
                ogar_install "$1" "$2"
                ;;
        update)
                ogar_update "$1" "$2"
                ;;
        uninstall)
                ogar_uninstall "$1" "$2"
                ;;
        "")
                echo "Blank sub-command. Please specify if you want to install, update or uninstall."
                exit 1
                ;;
        *)
                echo "Invalid sub-command. Please specify if you want to install, update or uninstall."
                exit 1
                ;;
esac
```

My install wasn't working.  After searching around here and there, I found that the arguments needed to be passed as function parameters.  After this, the install worked properly.

**Comment by SimonOrJ** [comment]
Included in pull request #600 .

**Comment by dl8sd11** [comment]
我糙你的

2016-05-26 14:16 GMT+08:00 SimonOrJ <notifications@github.com>:
